envisage/ui/action/tests/test_action_manager_builder.py

```diff
@@ -1,13 +1,8 @@
 """ Tests for the action manager builder. """
 
-try:
-    from pyface.api import GUI  # noqa
-except ImportError:
-    from nose.plugins.skip import SkipTest
-    raise SkipTest('Skipping due to missing GUI toolkit')
-
 # Enthought library imports.
 from envisage.ui.action.api import Action, ActionSet, Group, Menu
+from pyface.api import GUI  # noqa
```

Comment:
From offline discussion: it looks as thought we shouldn't need this line at all.